staging/src/k8s.io/client-go/tools/cache/delta_fifo.go

```diff
@@ -99,7 +99,7 @@ type DeltaFIFOOptions struct {
 	EmitDeltaTypeReplaced bool
 }
 
-// NewDeltaFIFOWithOptions returns a Store which can be used process changes to
+// NewDeltaFIFOWithOptions returns a Queue which can be used to process changes to
```

Comment:
A `Queue` is indeed a `Store`. But I think `DeltaFIFO` is modeled as a `Queue`, as enforced in code.

```
_ = Queue(&DeltaFIFO{}) // DeltaFIFO is a Queue
```

Reply:
Yeah sorry I thought I deleted that comment before I sent the review but I'm failing at github today apparently!

Reply:
You did! I read your comment from email.:P